IsraelHiking.Web/src/application/services/route-statistics.service.ts

```diff
@@ -107,7 +107,10 @@ export class RouteStatisticsService {
 
         // smooth the line in order to better calculate gain and loss:
         // changing x from Km to Km * 100 to better align with required altitude sensitivity
-        let simplifiedCoordinates = SpatialService.simplify(routeStatistics.points.map(p => [p.coordinate[0] * 100, p.coordinate[1]]), 5);
+        let pts = resample(routeStatistics.points.map(p=>p.coordinate), { dist: 0.025 }, false);
```

Comment:
I guess the comment above this method is inaccurate, can you please change it. Or remove it if you think the code is clear enough.